app/v1/services/CreatePeriodSummaryService.scala

```diff
@@ -30,16 +31,18 @@ import scala.concurrent.{ExecutionContext, Future}
 @Singleton
 class CreatePeriodSummaryService @Inject() (connector: CreatePeriodSummaryConnector) extends BaseService {
 
-  def createPeriodicSummary(
-      request: CreatePeriodSummaryRequest)(implicit ctx: RequestContext, ec: ExecutionContext): Future[CreatePeriodSummaryServiceOutcome] = {
-    val result = for {
-      responseWrapper <- EitherT(connector.createPeriodicSummary(request)).leftMap(mapDownstreamErrors(downstreamErrorMap))
-    } yield responseWrapper.map { _ =>
+  def createPeriodSummary(request: CreatePeriodSummaryRequest)(implicit
+      ctx: RequestContext,
+      ec: ExecutionContext): Future[ServiceOutcome[CreatePeriodSummaryResponse]] = {
+
+    def createSummaryResponse(wrapper: ResponseWrapper[Unit]): ResponseWrapper[CreatePeriodSummaryResponse] = {
       import request.body.periodDates._
-      CreatePeriodSummaryResponse(s"${periodStartDate}_$periodEndDate")
+      wrapper.copy(responseData = CreatePeriodSummaryResponse(s"${periodStartDate}_$periodEndDate"))
     }
 
-    result.value
+    connector
+      .createPeriodicSummary(request)
```

Comment:
for consistency with the endpoint name can we rename this method on the connector to `createPeriodSummary` (instead of `createPeriodicSummary`) (for both v1 and v2)